docs/reference/indices/rollover-index.asciidoc

```diff
@@ -42,19 +42,39 @@ NOTE: To roll over an index, a condition must be met *when you call the API*.
 automatically roll over indices when a condition is met, you can use {es}'s
 <<index-lifecycle-management, index lifecycle management (ILM) policies>>.
 
-The API accepts a single alias name and a list of `conditions`. The alias must point to a write index for
-a Rollover request to be valid. There are two ways this can be achieved, and depending on the configuration, the
-alias metadata will be updated differently. The two scenarios are as follows:
-
- - The alias only points to a single index with `is_write_index` not configured (defaults to `null`).
-
-In this scenario, the original index will have their rollover alias will be added to the newly created index, and removed
-from the original (rolled-over) index.
-
- - The alias points to one or more indices with `is_write_index` set to `true` on the index to be rolled over (the write index).
-
-In this scenario, the write index will have its rollover alias' `is_write_index` set to `false`, while the newly created index
-will now have the rollover alias pointing to it as the write index with `is_write_index` as `true`.
+The rollover index API accepts a single alias name
+and a list of `conditions`.
+To request a rollover,
+the specified alias must point to a write index,
+indicated by the `is_write_index` value.
```

Comment:
I'm a little concerned that readers may think `a write index` is some special kind of index, although I'm not sure how to clarify it.

I think `flag` instead of `value` might be a better phrasing, to indicate that it doesn't name a write index (i.e. it is not `is_write_index: my_index-000001`).

Reply:
Thanks for the feedback @gwbrown! Not picky at all. These are great points.

I've added 3094281 to address.

The wording for the 'write index' is a bit clunkier here, but I also feel it's now more specific.

```
To request a rollover,
the specified alias must point to an index
with an `is_write_index` flag
that is unconfigured (`null`) or `true`.
```